### PR TITLE
Fixes #64 - Creates a scheduler script requesting data for a specific milestone

### LINKED ISSUE #64: Add a Scheduler to the project
We need to add a scheduler to populate the data into the DB which is already there.
In #20 we had code. We will recreate a scheduler.

This issue is not about saving needsdiagnosis data into the DB, but just having the scheduler ready and see how it goes. 

* [x] Install a scheduler addon - Heroku Scheduler?
* [x] Create a `bin/`
* [x] Create a minimal script doing something dumb for proof of concept. (see #77 )
* [ ] Configure a task in the scheduler

### PR DESCRIPTION
**Pull request by karlcow**
This can be used on the command line with 

```bash
python ./bin/get_count.py sitewait
# MILESTONE sitewait: 2019-01-24T06:29:14Z 74
```
but it is meant to return the count of issues for a given milestone at the time of the request. It is meant to work in concert with a scheduler. For now we will be using Heroku Scheduler.

For now it will just print data on the heroku console.

Next stage, will be to modify the script to feed data to a DB.

### CODE REVIEW

File: bin/get_count.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import sys
+from urllib.parse import urljoin
+from urllib.request import Request
+from urllib.request import urlopen
+
+
+# Config
+URL_REPO = 'https://api.github.com/repos/webcompat/web-bugs/milestones/'
+MILESTONES = {
+    'needstriage': (1, 'open'),
```
[the remark this answers is not in the document]
Comment (karlcow) [reply bin/get_count.py]: It's why reviews are useful. Thanks for this I will fix it. 🙈

File: bin/get_count.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Get count of milestones data.
+"""
+
+import datetime
+import json
+import sys
+from urllib.parse import urljoin
+from urllib.request import Request
```
[the remark this answers is not in the document]
Comment (karlcow) [reply bin/get_count.py]: I could. One less dependency. And it's not very complicated for just a simple request.

**Comment by karlcow** [comment]
@laghee changes are done. :) Tell me if you approve it.

**Comment by laghee** [comment]
LGTM! 👍

**Pull request merged**